test/e2e/common/node/lifecycle_hook.go

```diff
@@ -130,7 +173,26 @@ var _ = SIGDescribe("Container Lifecycle Hook", func() {
 			testPodWithHook(podWithHook)
 		})
 		/*
-			Release: v1.9
+			Release : v1.19
+			Testname: Pod Lifecycle, poststart https hook
+			Description: When a post-start handler is specified in the container lifecycle using a 'HttpGet' action, then the handler MUST be invoked before the container is terminated. A server pod is created that will serve https requests, create a second pod with a container lifecycle specifying a post-start that invokes the server pod to validate that the post-start is executed.
+		*/
+		ginkgo.It("should execute poststart https hook properly [NodeConformance]", func() {
```

Comment:
I don't think this can be in conformance yet. Existing clusters that have to transition non-disruptively will likely have to turn off the feature gate for at least one release, and with it off, will fail this e2e test (this is similar to the ExecTimeout support and e2e test in 1.20) 

also, this has a minimum kubelet version required, which needs to be reflected in the test tag